### DISCUSSION TITLE
Question about OSC and Qlab

### DISCUSSION THREAD

**Discussion by jboHH** [cpvalente/ontime#1050] [Q&A] [answered]
Hi, 
I try to send osc commands from Ontime to qlab, which works in general very well. 
But I can´t find out how to send an osc command only for one specific event out of Ontime. So for example: If I start cue x in Ontime, at the same time via osc qlab should start a song.
I know the qlab osc command (which works) and I know how to send the osc command for each start of any event in Ontime. But not for a specific event. 
Anybody how can help me?

Thanks in advance
Jens

**Comment by alex-Arc** [comment, marked as answer]
Hey
Your observation is correct there is no way to to send osc from only specific events.
Maybe you can solve it by creating a custom field, it could be called fx `QLab`
and add an osc integration `On Start` with `/cue/{{eventNow.custom.qlab}}/start`

**Reply by jboHH** [reply]
Thanks a lot for your help!
